docs/provisioner/08-01-provisioning-gcp.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Provision clusters on Google Cloud Platform (GCP)
+type: Tutorials
+---
+
+This tutorial shows how to provision clusters with Kyma Runtimes (Runtimes) on Google Cloud Platform (GCP).
+
+## Prerequisites
+
+- Existing project on GCP
+- Service account on GCP with the following roles:
+    * Compute Admin
+    * Kubernetes Engine Admin
+    * Kubernetes Engine Cluster Admin
+    * Service Account User
+- Key generated for your service account downloaded in the `json` format
+- Secret from the service account key created in the `compass-system` Namespace with the key `credentials` and the value encoded with base64
+
+## Provision Kyma Runtime on GCP
+
+To provision Kyma Runtime, make a call to the Runtime Provisioner with this example mutation:
+
+> **NOTE:** To access the Runtime Provisioner, make a call from another Pod in the cluster containing the Runtime Provisioner or forward the port on which the GraphQL Server is listening.
+
```

Comment:
These two notes next to each other look a bit weird. Maybe you could take some info from the above note to the description above? or if these are 2 separate steps write them as such?